docs/algorithms/kem/kyber.yml

```diff
@@ -17,15 +17,9 @@ website: https://pq-crystals.org/
 nist-round: 3
 spec-version: NIST Round 3 submission
```

Comment:
Looks fishy to me: No change to spec version but different KATs?

Reply:
updated in 743c15ceea63ac88631d1ec16bab1ea6061eb6f0